### PR TITLE
revise install guide

### PR DESCRIPTION
**Pull request by geoffcline**
### Description

I've revised the install guide. I have removed the large number of admonitions (e.g. note, warning). I rearranged some content to improve flow. 

### Checklist
- [x] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Comment by geoffcline** [comment]
I've rebased and revised this @kishorj -- It's ready for review. 👍

File: docs/deploy/installation.md [hunk 1]
```diff
@@ -1,72 +1,74 @@
...
-    OR restrict access to the security groups tagged for the particular k8s cluster. Replace CLUSTER-NAME with the name of your k8s cluster and add it to the above IAM permissions.
-    ```
-        "Condition": {
-            "Null": {
-                "aws:ResourceTag/kubernetes.io/cluster/CLUSTER-NAME": "false"
-            }
+    }
+```
+
+Example condition for resource tag:
```
Comment (kishorj) [inline docs/deploy/installation.md:64]: Example condition for cluster name resource tag

**Review by kishorj — APPROVED** [review]

**Pull request merged**